Sources/XcbeautifyLib/Array+Safe.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Array+Safe.swift
+//  
+//
+//  Created by Charles Pisciotta on 6/19/23.
+//
+
```

Comment:
I'd remove these comments since it's information already tracked at the git level.